_attributes/common-attributes.adoc

```diff
@@ -42,6 +42,7 @@ endif::openshift-origin[]
 :insights-advisor-url: link:https://console.redhat.com/openshift/insights/advisor/[Insights Advisor]
 :hybrid-console: Red Hat Hybrid Cloud Console
 :hybrid-console-second: Hybrid Cloud Console
+:oc-first: OpenShift CLI (`oc`)
```

Comment:
It occurred to me last night that this should probably be:
```suggestion
:oc-first: the OpenShift CLI (`oc`)
```
to make usage more "normal", e.g.:
> You have installed {oc-first}

Reply:
I would argue against including "the". I don't think it's unusual to have to include "the" in front of an attribute (I do it all the time with operator names). And there might actually be some use cases where "OpenShift CLI" doesn't have a "the" in front.

Reply:
ah, ok - I don't use any like that and wasn't aware of the precedent. If it's a thing we already do, then I'll leave the `the` out of it :nerd_face:

Reply:
Yeah plus it could technically be capitalized "The" if it was at the beginning of the sentence :smile: